poncho/src/poncho/package_create.py

```diff
@@ -24,6 +24,20 @@
 
 conda_exec = 'conda'
 
+def create_env_from_dict(spec, conda_executable=None, download_micromamba=False, ignore_editable_packages=False):
```

Comment:
@BarrySlyDelgado Giving that pack_env transforms the json file again to a dict, wouldn't it be better to have `create_env_from_dict` as the "main" function? Then the one that uses json can json.loads it and pass it to the new `create_env_from_dict`.

Reply:
done.